docs/cli/geo/maps.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Maps
+description: Use Amplify CLI to create and manage maps to visualize GeoSpatial data in your app.
+---
+
+Amplify CLI's `geo` category enables you to create and manage map resources used to visualize GeoSpatial data in your application.
+You need to install Amplify CLI with the `@geo` tag in order to get the Geo functionality. You can use the following command to install this version globally.
```

Comment:
Say that Geo is in developer preview. Therefore, you need to install...

Reply:
Added "Since the `geo` category is in developer preview, you need to install...."